### DISCUSSION TITLE
Allow spaces in usernames

### DISCUSSION THREAD

**Discussion by Nv7-GitHub** [pocketbase/pocketbase#1722] [Q&A] [answered]
How do I allow users to put spaces in their usernames? (Using the `users` collection)

**Comment by ganigeorgiev** [comment, marked as answer]
This is not possible at the moment.

Usernames typically don't contain spaces because it could break for example a tagging/mention functionality, but I'll consider in a future release to expose the username regex as auth collection option so that developers can customize it however they want.
